src/TLuaInterpreter.cpp

```diff
@@ -10375,6 +10375,39 @@ int TLuaInterpreter::getModuleSync(lua_State* L)
     }
 }
 
+
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getPackages
+int TLuaInterpreter::getPackages(lua_State* L)
+{
+    Host& host = getHostFromLua(L);
+    auto packages = host.mInstalledPackages;
+    lua_newtable(L);
+    for (int i = 0; i < packages.size(); i++) {
+        lua_pushnumber(L, i + 1);
+        lua_pushstring(L, packages.at(i).toUtf8().constData());
+        lua_settable(L, -3);
+    }
+    return 1;
+}
+
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getModules
+int TLuaInterpreter::getModules(lua_State* L)
+{
+    Host& host = getHostFromLua(L);
+    auto modules = host.mInstalledModules;
+    int counter = 1;
+    QMap<QString, QStringList>::const_iterator iter = modules.constBegin();
+    lua_newtable(L);
+    while (iter != modules.constEnd()) {
+        lua_pushnumber(L, counter);
+        lua_pushstring(L, iter.key().toUtf8().constData());
+        lua_settable(L, -3);
+        counter++;
```

Comment:
This is a teeny bit inefficient. You'd be better to start the value at `0` and then just use the (more efficient)
```cpp
lua_pushnumber(L, ++counter);
```
three lines back, then you do not need this line at all...